akka-http-core/src/main/scala/akka/http/impl/settings/RoutingSettingsImpl.scala

```diff
@@ -28,6 +30,6 @@ object RoutingSettingsImpl extends SettingsCompanion[RoutingSettingsImpl]("akka.
     c getInt "range-count-limit",
     c getBytes "range-coalescing-threshold",
     c getIntBytes "decode-max-bytes-per-chunk",
-    c getString "file-io-dispatcher")
-
-}
+    c getString "file-io-dispatcher",
+    ErrorLoggingVerbosity(c getString "error-logging-verbosity"))
```

Comment:
Ah you mean `ParserSettings`. All right, I agree. It is worth noting RoutingSettings and ServerSettings do exist as well. Those two seems better candidates for more general `ErrorLoggingVerbosity`.